config/http.conf

```
@@ -51,9 +51,16 @@ http {
     ''      $request_id;
   }
 
+  # See https://www.nginx.com/blog/websocket-nginx
+  map $http_upgrade $connection_upgrade {
```

Nit-pick. I'd gotten into the habit of naming 'proxied' variables with a `proxy_` suffix (see configs above) and with the name to match it's use case. Could we do that here too? So that `$http_upgrade` is mapped to `$proxy_connection` ... because it's the now the "connection header on the proxy just mapped out". Makes it easy to know if it was a native or user defined variable at a glance ... and also what it is and where it's used.

```suggestion
  map $http_upgrade $proxy_connection {
```